build-support/bin/release.sh

```diff
@@ -773,7 +815,8 @@ while getopts "hdntcloepw" opt; do
     l) list_packages ; exit $? ;;
     o) list_owners ; exit $? ;;
     e) fetch_and_check_prebuilt_wheels ; exit $? ;;
-    p) build_pex ; exit $? ;;
+    p) build_pex fetch ; exit $? ;;
+    q) build_pex build ; exit $? ;;
```

Comment:
github won't let me comment directly on it, but: should update the help above re: "mutually exclusive".